packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js

```diff
@@ -124,16 +126,32 @@ MeetingInfoUtil.isPhoneNumber = (phoneNumber) => {
 };
 
 MeetingInfoUtil.getHydraId = (destination) => {
```

Comment:
jsdoc please

Reply:
we are not doing jsdocs on internal util files

Reply:
![img](https://media3.giphy.com/media/vX9WcCiWwUF7G/giphy.gif)

Reply:
wont do